### PR TITLE
Improve page headings

### PR DESCRIPTION
**Pull request by anpa**
This PR refactors some pages to ensure:

- each page has an H1 level heading;
- each page does not skip heading levels;

This is part of an effort to improve accessibility in the theme.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -2514,17 +2524,11 @@ ul {
   }
 }
 
-.post-sidebar h5 {
+.post-sidebar-title {
+  font-size: 18px;
   font-weight: 600;
 }
 
-@media (min-width: 1024px) {
```
Comment (yara-simargil) [inline style.css]: Why don't we need these border anymore?

File: styles/_post.scss [hunk 2]
```diff
@@ -139,12 +139,8 @@
     padding: 30px 0;
     text-align: center;
 
-    h5 {
-      @include desktop {
```
Comment (yara-simargil) [inline styles/_post.scss]: And this one as well, why can we remove it?

File: style.css (hunk 1 above)
Comment (anpa) [reply style.css]: It wasn't being used as there was no `h5` inside `.post-sidebar`

Comment (yara-simargil) [reply style.css]: Ah, I see. 👍

**Pull request merged**